.gitignore

```diff
@@ -0,0 +1,6 @@
+.idea
+target
+kickstarter.iml
+pom.xml
+src/main/webapp
+src/main/recources
```

Comment:
src/main/resources should not be ignored not sure about what's in src/main/webapp :)